Assets/Tests/PlayMode/Utils/Utils.cs

```diff
@@ -107,5 +111,61 @@ public void ClampFloatTest()
             //False above max positive
             Assert.AreEqual(10f, 11f.Clamp(-10f, 10f));
         }
+
```

Comment:
Commentaire à ajouter
```suggestion
/// <summary>
/// TODO
/// </summary>
```